django/thunderstore/core/tests/test_management_commands.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import override_settings
+
+from thunderstore.repository.models import Package, PackageVersion, Team
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("debug", (False, True))
+def test_create_test_data_debug_check(debug):
+    @override_settings(DEBUG=debug)
+    def test_debug():
+        try:
+            call_command("create_test_data", 10)
+            assert len(Package.objects.filter(name__icontains="Test_Package")) == 10
+            assert len(Team.objects.filter(name__icontains="Test_Team")) == 10
+            assert (
+                len(PackageVersion.objects.filter(name__icontains="Test_Package")) == 30
+            )
+        except CommandError as e:
+            assert "Only executable in debug environments" == str(e)
+
+    test_debug()
+
+
+@pytest.mark.django_db
+@override_settings(DEBUG=True)
+def test_create_test_data_clear():
+    team = Team(name="to_be_deleted_team")
+    team.save()
+    package = Package(name="to_be_deleted_package", owner=team)
+    package.save()
+    pv = PackageVersion(
+        name="to_be_deleted_package",
+        package=package,
+        version_number="1.0.0",
+        website_url="https://example.org",
+        description="Example mod",
+        readme="# This is an example mod",
+        file_size=5242880,
+    )
+    pv.save()
+
+    call_command("create_test_data", 10, "--clear")
+    assert len(Team.objects.filter(name="to_be_deleted_team")) == 0
```

Comment:
instead of copy-pasting, use `team.name`. Same for all of the below queries checking for deletion.

In fact, you could simply do `Team.objects.filter(pk=team.pk).exists()` for all of the objects you're checking deletion of.